lib/omnibus/packager.rb

```diff
@@ -68,7 +68,15 @@ def for_current_system
         family = "solaris"
       end
       if klass = PLATFORM_PACKAGER_MAP[family]
-        klass.is_a?(Array) ? klass : [ klass ]
+        package_types = klass.is_a?(Array) ? klass : [ klass ]
+
+        if package_types.include?(APPX) &&
+          !Chef::Sugar::Constraints::Version.new(version).satisfies?('>= 6.2')
```

Comment:
@mwrock Totally agree but we need to get Travis and Appveyor green before merging.